app/controllers/admin/edition_workflow_controller.rb

```diff
@@ -293,6 +297,8 @@ def action_name_as_human_interaction(action_name)
       "unpublish this edition"
     when "confirm_force_publish"
       "force publish this edition"
+    when "confirm_publish"
+      "Publish this edition"
```

Comment:
Should this be lowercase, as per the other action names?

```suggestion
      "publish this edition"
```